hack/make-rules/make-help.sh

```diff
@@ -21,22 +21,21 @@ set -o pipefail
 readonly red=$(tput setaf 1)
 readonly reset=$(tput sgr0)
 
-KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../..
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/../..
 ALL_TARGETS=$(make -C "${KUBE_ROOT}" PRINT_HELP=y -rpn | sed -n -e '/^$/ { n ; /^[^ .#][^ ]*:/ { s/:.*$// ; p ; } ; }' | sort)
-CMD_TARGETS=$(ls -l "${KUBE_ROOT}/cmd" |awk '/^d/ {print $NF}')
+CMD_TARGETS=$(cd "${KUBE_ROOT}/cmd"; find . -mindepth 1 -maxdepth 1 -type d | cut -c 3-)
```

Comment:
with gnu-find you get a warning if `-mindepth 1 -maxdepth1` (options) is not before `-type d` (find rules). updated